MYabrv.bib

```diff
@@ -48,31 +48,33 @@ @String{ICSM
 @String{ICSME = "Proc.\ Int'l Conf.\ on Software Maintenance and Evolution (ICSME)"}
 @String{ICSR = "Proc.\ Int'l Conf.\ on Software Reuse (ICSR)"}
 @String{ICST = "Proc.\ Int'l Conf.\ on Software Testing, Verification and Validation (ICST)"}
-@String{ICTCS	= "Proc.\ Italian Conf.\ on Theoretical Computer Science (ICTCS)"},
+@String{ICTCS = "Proc.\ Italian Conf.\ on Theoretical Computer Science (ICTCS)"},
 @String{ICTSS = "Proc.\ Int'l Conf.\ on Testing Software and Systems (ICTSS)"}
 @String{IDPT = "Proc.\ World Conf.\ on Integrated Design and Process Technology (IDPT)"}
 @String{IFAC = "World Congress of the International Federation of Automatic Control (IFAC)"}
 @String{iFM = "Proc.\ World Conf.\ on Integrated Formal Methods (iFM)"}
-@String{IPISE = "Intentional Perspectives on Information Systems Engineering"}
+@String{IPISE = "Intentional Perspectives on Information Systems Engineering (IPISE)"}
 @String{Informatik = "Jahrestagung der Gesellschaft f{\"{u}}r Informatik"}
 @String{ITNG = "Proc.\ Inte'l Conf.\ on Information Technology: New Generations (ITNG)"}
 @String{ITP = "Proc.\ Inte'l Conf.\ on Interactive Theorem Proving (ITP)"}
 @String{JMLC = "Proc.\ Joint Modular Languages Conference (JMLC)"}
+@String{MSR = "Proc.\ Working Conf.\ on Mining Software Repositories (MSR)"}
 @String{NaBIC = "World Congress on Nature Biologically Inspired Computing (NaBIC)"}
-@String{NATOSE = "Proc.\ NATO Conf.\ on Software Engineering"}
+@String{NATOSE = "Proc.\ NATO Conf.\ on Software Engineering (NATO SE)"}
 @String{NFM = "Proc.\ Int'l Conf.\ NASA Formal Methods (NFM)"}
 @String{MODELS = "Proc.\ Int'l Conf.\ on Model Driven Engineering Languages and Systems (MODELS)"}
 @String{Modellierung = "Proc.\ Modellierung"}
 @String{MODELSWARD = "Proc.\ Int'l Conf.\ on Models in Software Engineering (MODELSWARD)"}
-@String{MSR = "Proc.\ Working Conf.\ on Mining Software Repositories (MSR)"}
-@String{NODe = "Revised Papers Int'l Conf.\ NetObjectDays on Objects, Components, Architectures, Services, and Applications for a Networked World"},
+@String{MODULARITY = "Companion Proc.\ of Inte'l Conf.\ on Modularity"}
```

Comment:
Comanion indicates that the paper you added was just a workshop or so.

Inte'l -> Int'l

Reply:
Ok thanks for the clarification! :)